src/app.ts

```diff
@@ -100,6 +100,15 @@ async function app() {
     return;
   }
 
+  const fhirR4Dependency = config.dependencies['hl7.fhir.r4.core'];
+  if (!(fhirR4Dependency && fhirR4Dependency === '4.0.1')) {
+    logger.error(
+      'The package.json must specify FHIR R4 as a dependency. Be sure to' +
+        ' add "hl7.fhir.r4.core": "4.0.1" to the dependencies list.'
+    );
+    program.help();
```

Comment:
I could so an argument for using `program.help()` here and below, but we don't use it in other places, like if there is no package.json or an invalid package.json. So I'm not necessarily sure it makes sense in this case. Feel free to convince me otherwise though.

Reply:
Actually, you're probably right that it should just `return` similar to those other two places. There's nothing wrong with how they're invoking the command, so the `help` information doesn't give them any additional useful information.